packages/client/katello-host-tools/katello-host-tools.spec

```diff
@@ -334,36 +325,23 @@ exit 0
 %endif
 
 %dir %{katello_libdir}/
-%{katello_libdir}/constants.py*
-%{katello_libdir}/enabled_report.py*
-%{katello_libdir}/packages.py*
-%{katello_libdir}/repos.py*
-%{katello_libdir}/uep.py*
-%{katello_libdir}/utils.py*
-%{katello_libdir}/__init__.py*
-
-%if %{dnf_install}
-%{katello_libdir}/__pycache__/constants.*
-%{katello_libdir}/__pycache__/enabled_report.*
-%{katello_libdir}/__pycache__/packages.*
-%{katello_libdir}/__pycache__/repos.*
-%{katello_libdir}/__pycache__/uep.*
-%{katello_libdir}/__pycache__/utils.*
-%{katello_libdir}/__pycache__/__init__.*
-%else
+%{katello_libdir}/
 %attr(750, root, root) %{_sbindir}/katello-package-upload
 %attr(750, root, root) %{_sbindir}/katello-enabled-repos-upload
 
-%{plugins_dir}/enabled_repos_upload.py*
-%{plugins_dir}/package_upload.py*
-%endif
-
 %if %{zypper_install}
 %dir %{_usr}/lib/zypp/
 %dir %{_usr}/lib/zypp/plugins/
 %dir %{_usr}/lib/zypp/plugins/commit/
 %endif
 
+%{python3_sitelib}/apt_plugins
+%{python3_sitelib}/dnf_plugins
+
+%{python3_sitelib}/yum-plugins
+%{python3_sitelib}/zypper_plugins
```

Comment:
I do think it's safe to exclude apt for all RPMs. Then based on the OS (version) to exclude some files. Zypper is SuSE, yum is EL < 8, dnf is EL >= 8.

Reply:
Yup, this list can probably be seen more as a 'todo' left for the PR.  This is what was in a pyp2rpm run that I did, Kind of a "here's all the stuff when you install that wasn't already caught by other places in the spec"